pkgs/by-name/ni/nixos-render-docs/src/nixos_render_docs/redirects.py

```diff
@@ -47,23 +47,49 @@ def __str__(self):
 
     If you moved content, add its new location as the first element of the redirects mapping.
     Please update doc/redirects.json or nixos/doc/manual/redirects.json!
-""") # TODO: automatically detect if you just missed adding a new location, and make a tool to do that for you
+""")
         if self.identifiers_without_redirects:
             error_messages.append(f"""
 Identifiers present in the source must have a mapping in the redirects file.
     - {"\n    - ".join(self.identifiers_without_redirects)}
 
     This can happen when an identifier was added or renamed.
-    Please update doc/redirects.json or nixos/doc/manual/redirects.json!
-""") # TODO: add tooling in the development shell to do that automatically and point to that command
+
+    Added new content?
+        redirects add-content <identifier> <path>
+
+    Moved existing content to a different output path?
+        redirects move-content <identifier> <path>
+
+    Renamed existing identifiers?
+        redirects rename-identifier <old-identifier> <new-identifier>
+
+    Removed content? Redirect to alternatives or relevant release notes.
+        redirects remove-and-redirect <identifier> <target-identifier>
+
+    Note that you need to run `nix-shell doc` or `nix-shell nixos/doc/manual` to be able to run this command.
+""")
         if self.orphan_identifiers:
             error_messages.append(f"""
 Keys of the redirects mapping must correspond to some identifier in the source.
     - {"\n    - ".join(self.orphan_identifiers)}
 
     This can happen when an identifier was removed or renamed.
-    Please update doc/redirects.json or nixos/doc/manual/redirects.json!
-""") # TODO: add tooling in the development shell to do that automatically and point to that command
+
+    Added new content?
+        redirects add-content <identifier> <path>
+
+    Moved existing content to a different output path?
+        redirects move-content <identifier> <path>
+
```

Comment:
Found an unspecified requirement: nrd will complain if you rename an identifier but don't also rename references. It would be *very* helpful if this helper tool also did something akin to

```bash
sed -i 's/<old>/<new>/g' **/*
```

but that needs a bit of smarts, because we have capture old identifiers correctly (e.g. naively inserting `foo` and `foo2` after already having renamed the "tag" location of `foo` but not the "ref" locations, will produce `foo22` at the "tag").

So this is not blocking for this PR.